### ISSUE TITLE
Migrate Heapster to use Kubernetes 1.3

### ISSUE THREAD

**Issue by piosz** [kubernetes/heapster#1233]
- [x] bump godeps #1252
- [x] change tests to run on 1.3

**Comment by piosz** [comment]
cc @mksalawa @mwielgus

**Comment by andyxning** [comment]
We should absolutely update cAdvisor dependency.

**Comment by lattwood** [comment]
Is this going to resolve the issue in #1237 ?

```
I0809 17:35:05.658704       1 rate_calculator.go:51] Skipping rates for namespace:default/pod:gumco-223788322-yeys9/container:gumco - different create time new:2016-08-09 17:08:03.148174048 +0000 UTC  old:2016-08-09 17:08:03.139173623 +0000 UTC
I0809 17:35:05.658751       1 rate_calculator.go:51] Skipping rates for namespace:kube-system/pod:influxdb-grafana-dnwdh - different create time new:2016-08-09 17:07:51.450595189 +0000 UTC  old:2016-08-09 17:07:51.458595589 +0000 UTC
I0809 17:35:05.658788       1 rate_calculator.go:51] Skipping rates for namespace:kube-system/pod:kubernetes-dashboard-3717423461-n00hf - different create time new:2016-08-09 17:07:52.062625845 +0000 UTC  old:2016-08-09 17:07:52.060625744 +0000 UTC
I0809 17:35:05.658801       1 rate_calculator.go:51] Skipping rates for namespace:default/pod:kube-lego-2628658575-nbegw - different create time new:2016-08-09 17:25:09.245173745 +0000 UTC  old:2016-08-09 17:25:09.255174511 +0000 UTC
I0809 17:35:05.658824       1 rate_calculator.go:51] Skipping rates for namespace:kube-system/pod:kube-controller-manager-ip-10-0-5-158.ec2.internal - different create time new:2016-08-09 17:07:53.482696741 +0000 UTC  old:2016-08-09 17:07:53.48469684 +0000 UTC
I0809 17:35:05.658905       1 rate_calculator.go:51] Skipping rates for namespace:default/pod:kube-lego-2628658575-nbegw/container:kube-lego - different create time new:2016-08-09 17:27:40.598496929 +0000 UTC  old:2016-08-09 17:27:40.586496204 +0000 UTC
I0809 17:35:05.658942       1 rate_calculator.go:51] Skipping rates for namespace:kube-system/pod:kube2iam-4ie1g/container:kube2iam - different create time new:2016-08-09 17:08:03.722201157 +0000 UTC  old:2016-08-09 17:08:03.732201629 +0000 UTC
I0809 17:35:05.658968       1 rate_calculator.go:51] Skipping rates for namespace:default/pod:nginx-1537048879-eo10z - different create time new:2016-08-09 17:07:53.049675347 +0000 UTC  old:2016-08-09 17:07:53.047675248 +0000 UTC
I0809 17:35:05.659031       1 rate_calculator.go:51] Skipping rates for namespace:kube-system/pod:heapster-1006675365-4psz8 - different create time new:2016-08-09 17:07:53.60270267 +0000 UTC  old:2016-08-09 17:07:53.623703708 +0000 UTC
I0809 17:35:05.659088       1 rate_calculator.go:51] Skipping rates for namespace:kube-system/pod:influxdb-grafana-dnwdh/container:influxdb - different create time new:2016-08-09 17:07:51.708608106 +0000 UTC  old:2016-08-09 17:07:51.718608607 +0000 UTC
I0809 17:35:05.659140       1 rate_calculator.go:51] Skipping rates for namespace:default/pod:gumco-223788322-yeys9 - different create time new:2016-08-09 17:07:51.76361086 +0000 UTC  old:2016-08-09 17:07:51.775611461 +0000 UTC
I0809 17:35:05.659153       1 rate_calculator.go:51] Skipping rates for namespace:kube-system/pod:kube2iam-4ie1g - different create time new:2016-08-09 17:07:52.665656111 +0000 UTC  old:2016-08-09 17:07:52.675656613 +0000 UTC
I0809 17:35:05.659221       1 rate_calculator.go:51] Skipping rates for namespace:kube-system/pod:kube-controller-manager-ip-10-0-5-158.ec2.internal/container:kube-controller-manager - different create time new:2016-08-09 17:07:53.829713887 +0000 UTC  old:2016-08-09 17:07:53.830713936 +0000 UTC
```

**Comment by piosz** [comment]
@andyxning I bumped cadvisor to `v0.23.2-79-gc6c06d4` in #1252. We're trying to keep this in sync with the version in Kubernetes.

@lattwood probably not. I posted an update on #1237

**Comment by andyxning** [comment]
@piosz that's ok and we should keep in sync with kubernets. 👍

**Comment by piosz** [comment]
fixed in #1282
